includes/class-wc-siftscience-admin.php

```diff
@@ -315,7 +315,7 @@ private function get_section_fields( $sub_section ) {
 							'min'      => 0,
 							'step'     => 1,
 							'css'      => 'width:75px;',
-							'desc_tip' => 'Orders less than this value will not be automatically sent to sift.',
+							'desc_tip' => 'Orders less than this value wont be automatically sent to sift.',
```

Comment:
won't

Reply:
OK i didn't know that  honestly most  commas 
isn't aren't not sure about those I'll fix it